variables.tf

```diff
@@ -320,11 +320,11 @@ variable "managed_node_group" {
 
 #-----------------------------------------------ASG-Schedule----------------------------------------------------------------
 
-variable "create_schedule" {
-  description = "Determines whether to create autoscaling group schedule or not"
-  type        = bool
-  default     = true
-}
+#variable "create_schedule" {
+#  description = "Determines whether to create autoscaling group schedule or not"
+#  type        = bool
+#  default     = true
+#}
```

Comment:
Keep this `create_schedule` variable, otherwise remove it if not in use.

Reply:
This is fixed